HelpSource/Classes/Done.schelp

```diff
@@ -24,14 +24,41 @@ list::
 ## link::Classes/Demand::
 ::
 
+section:: Actions
+A number of UGens implement doneActions. These allow one to optionally free or pause the enclosing synth and other related nodes when the UGen is finished. You can use the constants in this class to name doneActions, which can be clearer than using the number codes alone. The available doneActions are as follows:
+
+table::
+## name     || value || description
+## doNothing || 0 || do nothing when the UGen is finished
```

Comment:
> Should we add idle?

Thanks for catching that, I actually used `idle` instead of `doNothing` in the .sc file, maybe I should use both?

> Or link to all the aliases actually?

Not sure what you mean.